inference-engine/tests/ie_test_utils/common_test_utils/data_utils.hpp

```diff
@@ -183,6 +183,39 @@ void inline  fill_data_random(InferenceEngine::Blob::Ptr &blob, const uint32_t r
     fill_data_random(rawBlobDataPtr, blob->size(), range, start_from, k, seed);
 }
 
+template<InferenceEngine::Precision::ePrecision PRC>
+void inline fill_random_unique_sequence(InferenceEngine::Blob::Ptr& blob,
+                                        uint32_t range,
+                                        int32_t start_from = 0,
+                                        const int32_t k = 1,
+                                        const int32_t seed = 1) {
+    using dataType = typename InferenceEngine::PrecisionTrait<PRC>::value_type;
```

Comment:
Can you write some comments for the function describing what it does?

Reply:
Done